### PR TITLE
Suppress escape sequences at end of hack/test-assets.sh

### PR DESCRIPTION
**Pull request by smarterclayton**
@liggitt [test]

### CODE REVIEW

File: hack/test-assets.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ pushd "${OS_ROOT}/assets" > /dev/null
   grunt test
   grunt build
 popd > /dev/null
-
+printf \\e[0m
```
Comment (liggitt) [inline hack/test-assets.sh]: `tput reset`?

**Comment by liggitt** [comment]
LGTM

**Comment by smarterclayton** [comment]
Didn't work

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[merge] cleared colors

**Pull request merged**